src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -45,6 +44,13 @@ public static void GetDirectoryName(string path, string expected)
             Assert.Equal(expected, Path.GetDirectoryName(path));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".Net Core returns null dotnet/corefx #19337")]
+        public static void GetDirectoryName_EmptyString_Throws()
+        {
+            Assert.Throws<ArgumentException>(() => Path.GetDirectoryName(string.Empty));
+        }
```

Comment:
Why aren't we instead fixing .NET Core to match the desktop (and explicitly documented) behavior of throwing for string.Empty?